experiments/AMIP/modular/coupler_driver_modular.jl

```diff
@@ -378,29 +390,41 @@ cs = CoupledSimulation{FT}(
 
 
 #=
-## Initial Component Model Exchange
+## Initialize Component Model Exchange
 =#
 
-## share states and fluxes between models
-# 1) import states into the coupler
+#Coupler needs; (sfc.T_sfc, atmos.ts) -> atmos.rho_sfc
+# sfc need: coupler.rho_sfc
+#Coupler needs; sfc.q_sfc
+# coupler needs q_sfc, sfc.surface_conditions, atmos_surface conditions for updates
+
+#Atmos needs; coupler.T_S (, coupler.albedo)
```

Comment:
should these both be combined as a tuple? (not a big deal since it's just a comment)

Reply:
This was actually some additional info I wrote before adding the comments below - I meant to rm this, since the comments should be sufficient. Thank you for spotting it!